src/styles/app/35-sidebar-nav.css

```
@@ -265,3 +265,41 @@
+	letter-spacing: 0.08em;
+	text-transform: uppercase;
+	line-height: 1;
+	pointer-events: none;
+	user-select: none;
+	white-space: nowrap;
+	box-shadow: var(--pill-shadow);
+}
+
+.sidebarAlphaBadge svg {
+	flex-shrink: 0;
+	color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)